src/game.h

```diff
@@ -411,7 +411,7 @@ class game
 
         void open_gate( const int examx, const int examy, const ter_id handle_type );
 
-        bionic_id random_good_bionic() const; // returns a non-faulty, valid bionic
+        std::string random_good_bionic() const; // returns a non-faulty, valid bionic
```

Comment:
I'm mot sure which functions you mean. `add_bionic` and similar stuff that is low-level, just modifying `player::my_bionics` could probably stay there. But most mutation related functions (that use the characters mutations map) are also separate in mutation.cpp and not in character.cpp, so this might be more consistent in the end.

I guess we need the advise of an authority on that.